microdnf/commands/advisory/advisory_info.cpp

```diff
@@ -75,14 +75,17 @@ void AdvisoryInfoCommand::run() {
     auto adv_q_updates = libdnf::advisory::AdvisoryQuery(ctx.base);
 
     if (installed->get_value() || all->get_value()) {
-        auto installed_package_query = package_query.filter_installed();
+        auto installed_package_query = package_query;
+        installed_package_query.filter_installed();
```

Comment:
Yes I know. The use of query filters was incorrect. That's why I made these changes to a special commit. And I provided a commit message with details. The message ends: "This commit fix the use of query filters, but it seems neecesary to revise the logic of the advisory commands."
I was afraid something would break. The original code is confusing and I wanted to discuss it with the author. I'm contacting @kontura.

Reply:
I see, yes I think I made an actual errors here and this commit fixes it.

I will make sure once I get to creating the CI tests for advisory.